From 9a1cbce209d268d3fc0c8e56ee953b310a204ded Mon Sep 17 00:00:00 2001
From: Andrea Piccinelli <frasten@gmail.com>
Date: Thu, 4 Oct 2012 09:40:13 +0300
Subject: [PATCH] http_loadtime: fixed stderr redirection with 'time'

---
 plugins/node.d/http_loadtime.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/plugins/node.d/http_loadtime.in b/plugins/node.d/http_loadtime.in
index bf17619..bb0675d 100644
--- a/plugins/node.d/http_loadtime.in
+++ b/plugins/node.d/http_loadtime.in
@@ -67,7 +67,7 @@ TMPDIR=`mktemp -d` || exit 1
 trap "rm -rf $TMPDIR" EXIT
 
 cd $TMPDIR || exit 1
-loadtime=$(time -p wget -p --no-cache --delete-after $target -q 2>&1 | awk '/^real / { print $2 }')
+loadtime=$((time -p wget -p --no-cache --delete-after $target -q) 2>&1 | awk '/^real / { print $2 }')
 cd ..
 
 echo "loadtime.value $loadtime"
-- 
1.7.2.5

