Index: node/node.d.linux/iostat_ios.in
===================================================================
RCS file: /cvsroot/munin/munin/node/node.d.linux/iostat_ios.in,v
retrieving revision 1.1.2.1
diff -u -r1.1.2.1 iostat_ios.in
--- node/node.d.linux/iostat_ios.in	4 Aug 2004 20:35:08 -0000	1.1.2.1
+++ node/node.d.linux/iostat_ios.in	9 Sep 2004 11:30:24 -0000
@@ -20,7 +20,6 @@
 use IO::File;
 
 use Storable qw(store retrieve);
-use Data::Dumper;
 
 # (c) 2004 - Per Andreas Buer
 # GPL v2 Yadda yadda.
